code-reviews/recipes/Markdown.md

```diff
@@ -0,0 +1,175 @@
+    - uses: actions/checkout@v2
+    - name: Use Node.js
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Run Markdownlint
+      run: |
+        npm i -g markdownlint-cli
+        markdownlint "**/*.md"
+```
+
+## Code Review Checklist
+
+Apart from the items on the [Code Review Checklist](../readme.md) you should also look for these markdown specific code review items
+
+- [ ] Is the document easy to read and understand and does it follow [good writing guidelines](#writing-style-guidelines)?
+- [ ] Is there a single source of truth or is content repeated in more than one document?
+- [ ] Is the documentation up to date with the code?
+
+## Writing Style Guidelines
+
+The following are some examples of writing style guidelines.
+
+Agree in your team which guidelines you should apply to your project documentation.
+Save your guidelines together with your documentation so they are easy to refer back to.
+
+### Wording
+
+- Use inclusive language, and avoid jargon and uncommon words. The docs should be easy to understand.
+- Be clear and consise, stick to the goal of the document.
+
+### Document Organization
+
+- Organize documents by topic rather than type, this makes it easier to find the documentation
+- Each folder should have a top-level readme.md and any other documents within that folder should link directly or indirectly from that readme.md
+- Document names with more than one word should use underscores instead of spaces, for example `machine_learning_pipeline_desgin.md`. The same applies to images.
+
+### Headings
+
+- Start with a H1 and respect the order H1 > H2 > H3 etc.
+- Avoid putting numbers in headings. Numbers shift, and can create outdated titles
+- Avoid using symbols and special characters in headers, this causes problems with anchor links
+- Avoid links in headers
+
+### Links
+
+- Avoid duplication of content, instead link to the `single source of truth`
+- Link but don't summarize. Summarizing content on another page leads to the content living in two places.
+- Use meaningful anchor texts, e.g. instead of writing `Follow the instructions [here](../recipes/Markdown.md)` write `Follow the [Markdown guidelines](../recipes/Markdown.md)`
+
+### Lists
+
+- List items should start with capital letters
+- Use ordered lists when the items describe a sequence to follow, otherwise use unordered lists
+- For ordered lists, prefix each item with `1.` When rendered, the list items will appear with sequential numbering. This avoids number-gaps in list
+- Do not add commas (,) or semicolons (;) to the end of list items, and avoid periods (.) unless the list item represents a complete sentence
+
+### Images
+
+- Place images in a separate directory named `img`
+- Name images appropriately, avoiding generic names like `screenshot.png`
+- Avoid adding large images or videos to source control, link to an external location instead.
+
+### Emphasis
+
+- Use **bold** or _italic_ or `tics` to emphasise.
```

Comment:
I suggest you to use backticks to provide code in the documentation. if it's an inline code piece, use single backticks, if it's multi-line codes, use triple backticks and if you're using triple backticks, always add language after the triple backticks (e.g. ```yaml)

Reply:
Fixed